packages/next-templates/src/app/klachtenformulier-inloggen/page.tsx

```diff
@@ -0,0 +1,110 @@
+'use client';
+
+import {
+  Article,
+  BreadcrumbNav,
+  BreadcrumbNavLink,
+  BreadcrumbNavSeparator,
+  ButtonLink,
+  Heading1,
+  Heading2,
+  Paragraph,
+  Page,
+  PageContent,
+} from '@utrecht/component-library-react';
+
+import { UtrechtDigidLogo, UtrechtEherkenningLogo } from '@utrecht/web-component-library-react';
+
+import { ExampleHeader } from '@/components/ExampleHeader/ExampleHeader';
+import { ExampleNavigation } from '@/components/ExampleNavigation/ExampleNavigation';
+import { ExampleFooter } from '@/components/ExampleFooter/ExampleFooter';
+import { PreFooterNav } from '@/components/PreFooterNav/PreFooterNav';
+import { IconChevronRight } from '@tabler/icons-react';
+
+import '@utrecht/design-tokens/dist/index.css';
+import '@/app/styling/css/detail-page.css';
+import '@/app/styling/css/productpagina-klachtenformulier.css';
+
+import { useEffect } from 'react';
+
+export default function home() {
+  const pageTitle = 'Klacht over de gemeente doorgeven';
+  const websiteLabel = 'gemeente Voorbeeld';
+
+  useEffect(() => {
+    if (typeof document?.title === 'string') {
+      document.title = `${pageTitle} - ${websiteLabel}`;
+    }
+  }, []);
+
+  return (
+    <Page>
+      <ExampleHeader />
+      <ExampleNavigation />
+
+      <BreadcrumbNav label="Kruimelpad" className="voorbeeld-breadcrumb-nav">
+        <BreadcrumbNavLink href="/" index={0} rel="home">
+          Home
+        </BreadcrumbNavLink>
+        <BreadcrumbNavSeparator>
+          <IconChevronRight className="icon-chevron-right" />
+        </BreadcrumbNavSeparator>
+        <BreadcrumbNavLink href="/wmebv" index={1} rel="contact">
+          Contact
+        </BreadcrumbNavLink>
+        <BreadcrumbNavSeparator>
+          <IconChevronRight className="icon-chevron-right" />
+        </BreadcrumbNavSeparator>
+        <BreadcrumbNavLink href="/productpagina-klachtenformulier" index={2} rel="up">
+          Producten
+        </BreadcrumbNavLink>
+        <BreadcrumbNavSeparator>
+          <IconChevronRight className="icon-chevron-right" />
+        </BreadcrumbNavSeparator>
+        <BreadcrumbNavLink href="/klachtenformulier-inloggen" index={3} rel="up">
+          Inloggen
+        </BreadcrumbNavLink>
+      </BreadcrumbNav>
+
+      <PageContent>
+        <Article id="main" className="voorbeeld-article-space">
+          <Heading1>{pageTitle}</Heading1>
+
+          <Heading2>Inloggen met DigiD</Heading2>
+          <Paragraph>Log in met uw DigiD om verder te gaan.</Paragraph>
+
+          <div className="voorbeeld-button-group">
```

Comment:
is niet echt een goede class name aangezien er maar 1 button in zit

Reply:
Fair, aangepast!